cmd/booster-bitswap/run.go

```diff
@@ -35,6 +36,16 @@ var runCmd = &cli.Command{
 			Usage:    "the endpoint for the boost API",
 			Required: true,
 		},
+		&cli.BoolFlag{
+			Name:  "tracing",
+			Usage: "enables tracing of booster-http calls",
+			Value: false,
+		},
+		&cli.StringFlag{
+			Name:  "tracing-endpoint",
+			Usage: "the endpoint for the tracing exporter",
+			Value: "http://tempo:14268/api/traces",
+		},
```

Comment:
Would it make sense to only have the "tracing-endpoint" flag? It's implied that tracing is turned on if this flag is there so maybe we don't need the "tracing" flag?

Reply:
I find it nice not having to specify the tracing-endpoint when running the devnet, although what you suggest also makes sense.